primer_designer/views.py

```diff
@@ -40,24 +39,16 @@ def index(request):
             # generate primers
             return create(request, regions_form)
         else:
-            error = ast.literal_eval(pp.pformat(regions_form.errors))
-
             messages.add_message(
                 request,
                 messages.ERROR,
-                mark_safe(
-                    f"Error in given primer design input: {error['regions'][0]}"
-                ),
```

Comment:
if there's an error with what they enter, does it return back to the user what was entered? does it show the error? (can't remember what it used to do...)

Reply:
Added their own primer input in error message. Error message appears as pop up at the top and also below the form TextArea
![Screenshot from 2022-06-18 21-17-46](https://user-images.githubusercontent.com/60819172/174455884-45385095-a94a-4a40-91a6-b6aeb8f17e45.png)